src/components/Footer.tsx

```diff
@@ -320,7 +320,7 @@ const Footer = ({ lastDeployLocaleTimestamp }: FooterProps) => {
         <Button
           variant="outline"
           isSecondary
-          onClick={() => scrollIntoView("__next")}
+          onClick={() => scrollIntoView("top")}
```

Comment:
This function currently accepts an `id` (`toId`), but we could broaden it's usage by refactoring to use a css selector string with `querySelector`

I would just update `scrollIntoView` to accept a query selector instead of an `id`, and then we can just target the `h1` in this case, and update any other instances passing an id to prefix a `#`

For example: 
```ts
// src/lib/utils/scrollIntoView.ts

export const scrollIntoView = (
  selector: string,
  options: ScrollIntoViewOptions = { behavior: "smooth", block: "start" }
): void => {
  const element = document.querySelector(selector)

  if (!element) return

  element.scrollIntoView(options)
}
```

Then this line becomes

```suggestion
          onClick={() => scrollIntoView("h1")}
```

And I believe the only other place we use this is in `Button`, where we can prefix the id with `#`:

```tsx
// src/components/ui/buttons/Button.tsx:108

      toId && scrollIntoView("#" + toId)
```

Reply:
Hi @wackerow , Thanks for the clear feedback! I've updated the implementation as suggested and just pushed the changes. Let me know if anything else needs adjustment.